pip/flatpak-pip-generator

```diff
@@ -192,28 +226,34 @@ modules = []
 vcs_modules = []
 sources = {}
 
-tempdir_prefix = 'pip-generator-{}'.format(output_filename.replace('.json', ''))
+tempdir_prefix = 'flatpak-pip-generator'
 with tempfile.TemporaryDirectory(prefix=tempdir_prefix) as tempdir:
-    pip_download = [
-        pip_executable,
+    pip_download = flatpak_cmd + [
```

Comment:
This effectively makes it impossible to run this script in flatpak sandbox. At least add an option to run without flatpak, and/or check if we are already in sandbox.

Reply:
I didn't consider that case, will do.